burnman/eos/mie_grueneisen_debye.py

```diff
@@ -174,6 +174,10 @@ def validate_parameters(self, params):
         """
         Check for existence and validity of the parameters
         """
+        if 'T_0' not in params:
+            params['T_0'] = 300.
+        if 'P_0' not in params:
+            params['P_0'] = 0.
```

Comment:
Yes, that's right. But as we don't currently run bm.validate_parameters() if we're using the SLB/MGD eos, I think this is the best place to initialise the parameter. Happy to hear alternatives though, if you have a better idea...